platform/infra/lib/infra/vpc.ts

```diff
@@ -9,8 +9,42 @@ export default class VpcStack extends Stack {
     constructor(scope: Construct, id: string, props: StackProps) {
         super(scope, id, props);
         this.vpc = new Vpc(this, "appmod-vpc", {
-            vpcName: "appmod-vpc"
+            vpcName: "appmod-vpc",
+            maxAzs: 3,
+            ipAddresses: IpAddresses.cidr('192.168.0.0/16'),
```

Comment:
Any reason why this CIDR vs 10.?

Reply:
Only for testing, but fixed to 10.0.0.0 in commit [4723a31](https://github.com/shapirov103/appmod-blueprints/pull/15/commits/4723a310ea37a1462eddbe39288ded3a9d4e544b)